types/Cargo.toml

```diff
@@ -25,7 +25,7 @@ derive_more = { version = "0.99.17", default-features = false, features = [ "fro
 hashbrown = { version = "0.12.3", default-features = false, features = [ "ahash", "inline-more" ] }
 hex = { version = "0.4.3", default-features = false, features = [ "alloc" ] }
 inx = { version = "1.0.0-beta.8", default-features = false, optional = true }
-iota-crypto = { version = "0.15.1", default-features = false, features = [ "ed25519", "blake2b", "ternary_encoding" ] }
+iota-crypto = { version = "0.15.2", default-features = false, features = [ "ed25519", "blake2b", "ternary_encoding" ] }
```

Comment:
i don't think it's necessary, as patches will always be fetched with the latest version, unless you clamp the version to eg. `=0.15.1`

Reply:
Right, but that's just the way we do :)